.husky/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd frontend
+echo frontend code is formatting...
+npx lint-staged
+
```

Comment:
Can we not leverage `npm run format`?

Reply:
yes @krishnaacharyaa we can do but it formatting every file.

Reply:
@Ameerjafar  Yeah, thats our end goal right? All the files need to be formatted. what does npx lint-staged do different? only the commitable files is it? if yes then fine that will make sense, having lint-staged.